highlight/highlight
# Fix log panel requests

## Summary
Decrease height to be able to see all the logs

## How did you test this change?
1) View a session
2) In the Highlight dev tools, click "Show in Log Viewer"
3) Scroll to the end of the logs
- [ ] All logs are displayed
- [ ] Can see "Loading" when loading more logs

## Are there any deployment considerations?
N/A

## Does this work require review from our design team?
N/A
